configure.ac

```diff
@@ -24,17 +25,30 @@ LT_PREREQ([2.4])
 LT_INIT()
 
 # Process user optons.
-
-# Does the user want to turn off all attempts to set fortran flags?
-# This is necessary to handle situations where the configure-provided
-# flags will be wrong for a particular install. With this setting, the
-# user is in complete control of Fortran flags.
-AC_MSG_CHECKING([whether we should attempt to set Fortran flags for the user])
-AC_ARG_ENABLE([fortran-flag-setting],
-  [AS_HELP_STRING([--disable-fortran-flag-setting],
-    [Disable any attempts by configure to set fortran flags. With this option, only user-provided flags (set in FCFLAGS) will be used compiling Fortran code.])])
-test "x$enable_fortran_flag_setting" = xno || enable_fortran_flag_setting=yes
-AC_MSG_RESULT([$enable_fortran_flag_setting])
+AC_ARG_ENABLE([mixed-mode],
+  [AS_HELP_STRING([--enable-mixed-mode],
+    [Build using mixed mode.  Enables both 64-bit and 32-bit reals in Fortran. This option will be ignored if --disable-fortran-flag-setting is also given.])])
+AS_IF([test ${enable_mixed_mode:-no} = no],
+  [enable_mixed_mode=no],
+  [enable_mixed_mode=yes])
+AC_ARG_WITH([mpi],
+  [AS_HELP_STRING([--with-mpi],
+    [Build with MPI support.  This option will be ignored if --disable-fortran-flag-setting is also given. (Default yes)])])
+AS_IF([test ${with_mpi:-yes} = yes],
+  [with_mpi=yes],
+  [with_mpi=no])
+AC_ARG_WITH([openmp],
```

Comment:
I think you want --enable-openmp here, not --with-openmp here.

Reply:
You are likely correct that `--enable-` is a better choice than `--with-`, especially since openMP is not something managed in a library separate from the compiler.  I can make this change in a new commit.